PayPalNativeCheckout/src/main/java/com/braintreepayments/api/PayPalNativeCheckoutVaultRequest.java

```diff
@@ -87,7 +88,7 @@ public int describeContents() {
         return 0;
     }
 
-    public static final Creator<PayPalNativeCheckoutVaultRequest> CREATOR = new Creator<PayPalNativeCheckoutVaultRequest>() {
+    public static final Creator<PayPalNativeCheckoutVaultRequest> CREATOR = new Creator<>() {
```

Comment:
Is this a breaking change? Java is far from my forte so not sure if removing `PayPalNativeCheckoutVaultRequest` from the `Creator` is considered a breaking change in java. If it is, we should call it out in the changelog.

Reply:
nah java will cast to the type declaration

Reply:
Nice! That's dope.